src/test/java/edu/cornell/kfs/concur/services/ConcurAccountValidationServiceTest.java

```diff
@@ -0,0 +1,444 @@
+package edu.cornell.kfs.concur.services;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kuali.kfs.sys.KFSConstants;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.businessobjects.ConcurAccountInfo;
+import edu.cornell.kfs.concur.businessobjects.ValidationResult;
+import edu.cornell.kfs.concur.service.impl.ConcurAccountValidationServiceImpl;
+import static org.junit.Assert.assertEquals;
+
+public class ConcurAccountValidationServiceTest {
+    
+    private ConcurAccountValidationServiceImpl concurAccountValidationService;
+    private ConcurAccountInfo concurAccountInfo;
+    
+    @Before
+    public void setUp() throws Exception {
+        concurAccountValidationService = new ConcurAccountValidationServiceImpl();
+        concurAccountValidationService.setAccountService(new MockAccountService());
+        concurAccountValidationService.setObjectCodeService(new MockObjectCodeService());
+        concurAccountValidationService.setSubObjectCodeService(new MockSubObjectCodeService());
+        concurAccountValidationService.setProjectCodeService(new MockProjectCodeService());
+        concurAccountValidationService.setSubAccountService(new MockSubAccountService());
+        concurAccountInfo = buildValidConcurAccountInfo();
+    }
+    
+    @After
+    public void testDown() {    
+        concurAccountInfo = null;
+    }
+    
+    private ConcurAccountInfo buildValidConcurAccountInfo(){
+        ConcurAccountInfo concurAccountInfo = new ConcurAccountInfo();
+        concurAccountInfo.setChart(ConcurAccountValidationTestConstants.VALID_CHART);
+        concurAccountInfo.setAccountNumber(ConcurAccountValidationTestConstants.VALID_ACCT_NBR);
+        concurAccountInfo.setSubAccountNumber(ConcurAccountValidationTestConstants.VALID_SUB_ACCT);
+        concurAccountInfo.setObjectCode(ConcurAccountValidationTestConstants.VALID_OBJ_CD);
+        concurAccountInfo.setSubObjectCode(ConcurAccountValidationTestConstants.VALID_SUB_OBJECT);
+        concurAccountInfo.setProjectCode(ConcurAccountValidationTestConstants.VALID_PROJECT_CODE);
+        return concurAccountInfo;
+    }
+    
+    @Test
+    public void isAccountValid(){
+        ValidationResult validationResult = concurAccountValidationService.checkAccount(ConcurAccountValidationTestConstants.VALID_CHART, ConcurAccountValidationTestConstants.VALID_ACCT_NBR);
+        assertEquals("Validation was expected to pass", true, validationResult.isValid());
+        assertEquals("No error messages expected", 0, validationResult.getMessages().size());    
+    }
+    
+    @Test
+    public void isAccountNull(){
+        ValidationResult validationResult = concurAccountValidationService.checkAccount(ConcurAccountValidationTestConstants.VALID_CHART, null);
+        assertEquals("Validation was expected to fail but returned true", false, validationResult.isValid());
```

Comment:
This is nitpicky, but you could use assertFalse instead of assertEquals for these.